packages/dropdownMenu/README.md

```diff
@@ -5,24 +5,33 @@ A dropdown toggles an overlay menu of actions. It appears as a result of a user
 Dropdown menus can be opened and navigated via cursor and keyboard.
 
 ## DropdownMenu triggers
+
 By default, use one of the dropdown button components (e.g.: `PrimaryDropdownButton`) as the dropdown trigger.
 
 If you're not using a button as your dropdown trigger, be sure whatever you use looks like something the user is supposed to interact with.
 
 ## Icons and avatars in items
+
 Dropdown menu items can display an icon or avatar before or after the menu item text. By default, the avatar or icon is sized at `iconSizeXs`.
 
-## Disabled menu items
-TODO: Collaborate with the design team to decide on some guidelines for disabled items. For example: when should we render a disabled item instead of not showing the item in the dropdown at all?
+## Menu item states
+
+**Enabled:** By default, menu items are enabled and will trigger some action.
+
+**Disabled:** Disable a menu item if it triggers an action that is not currently possible to execute, but could become enabled by an action the user can take. Show a tooltip if the user might require more information about why that menu item is disabled.
+
+**Hidden:** Hide a menu item if there is no way a user can execute the action triggered by that menu item. If all menu items in a DropdownMenu are hidden, do not show the trigger that opens the menu when clicked.
 
```

Comment:
we need a section in this document to call out ordering of actions in menu

View 
Edit
-separator-
Delete

View 
Edit
-separator-
Other actions 
-separator-
Other destructive actions 
Delete
 
In appropriate place we should also indicate that destructive actions like Delete and Remove should be red (perhaps utility class for this so more semantic at dev time)

Reply:
These rules fall outside the scope of this ticket. I'll be sure we cover this in another ticket for documenting design guidelines.

Reply:
SG ill keep a reference to this comment too.